data/sources/blocks/e-commerce/invoicing-block-60.yaml

```diff
@@ -0,0 +1,56 @@
+id: invoicing-block-60
+questions:
+- answers:
+  - description: ''
+    id: invoicing-answer-60
+    label: ''
+    mandatory: true
+    options:
+    - label: 'Yes'
+      value: 'Yes'
+    - label: 'No'
+      value: 'No'
+    type: Radio
+    guidance:
+      show_guidance: Show Invoices, EDI, UBL and XML Definitions
+      hide_guidance: Hide Invoices, EDI, UBL and XML Definitions
+      content:
+      - title: Invoices
+        description: "There are invoices in paper form and electronic form.\
+         \ Invoices in electronic form are of two types:</br>E-invoices in a standard structure suitable for\
+         \ automated processing (for example EDI, UBL, XML), excluding the transmission of PDF files.\
+         \ They are exchanged either directly or via service operators or via an electronic banking system.\
+         \ </br>Invoices in electronic form not suitable for automated processing, for example e-mails,\
```

Comment:
There shouldn't be `<br>` tags in guidance content - use another content block with just a description